pom.xml

```diff
@@ -18,11 +18,11 @@
     <parent>
         <groupId>org.jberet</groupId>
         <artifactId>jberet-parent</artifactId>
-        <version>1.4.5.Final</version>
+	<version>1.4.7.Final</version>
     </parent>
 
     <artifactId>jberet-support</artifactId>
-    <version>1.4.1.Final-SNAPSHOT</version>
+    <version>1.4.7.Final-SNAPSHOT</version>
```

Comment:
I think we should keep the sequence, and not skip directly to 1.4.7 for jberet-support version. So this line need no change.

Reply:
Now it should be ok.